apis/PostsEndpoint.js

```diff
@@ -85,9 +85,9 @@ export default {
 
   post(apiOptions, data) {
     const path = 'api/submit';
-    const body = formatPostData(data);
+    const query = formatPostData(data);
```

Comment:
I broke this in my last patch. This is (very confusingly) meant to be query parameters. So much disappoint.